app/src/db/migrations/20210607111302_023-farmopscreening-form-version-data.js

```diff
@@ -0,0 +1,37 @@
+const PREFIX = require('../../forms/attestations/farmopscreening/constants').PREFIX;
+const CREATED_BY = 'migration-023';
+
+exports.up = function(knex) {
+  return Promise.resolve()
+    // delete version 2
+    .then(()=> {
+      return knex(`${PREFIX}_form_version`)
+        .where('formVersionId', 2)
+        .del();
+    })
+    // create version 2
+    .then(() => {
+      // get formId
+      const form1 = knex.select('formId')
+        .from(`${PREFIX}_form`)
+        .orderBy('createdAt', 'asc')
+        .limit(1);
```

Comment:
This part not neccessary really either since there would only be 1 record in for each form, but it's a fine thing to leave in queries like this, though usually would order descending in these cases since you'd want the guard to take the most recent record I think. 
But no change needed since it's only ever going to have 1